packages/client/hmi-client/src/components/model-template/tera-model-template.vue

```diff
@@ -51,6 +52,7 @@ const props = defineProps<{
 	model: any;
 	isEditable: boolean;
 	isDecomposed: boolean;
+	id?: string;
```

Comment:
`portId` could be shared among different cards. The `portId` isn't a uuid, it is actually the name of the state. So the card that holds `S -> I` would hold the port ids `S` and `I`. Another one can hold `I` and `R`. So the cardId + portId makes it unique. The `id` is optional since the cards that are rendered in the sidebar for dragging and dropping don't have any.

Reply:
Not a fan of optional parameters, probably a better approach is to pass in dummy-strings for places where you don't actually need the ids, that way you have a deterministic way of dealing with things down stream. But okay for now.